### PR TITLE
Feat: For <select> set .chosen & apply chosen

### PR DESCRIPTION
**Pull request by IgorA100**
If more 10 options in <selector>, search is turned on, if less 10 options, the search is not turned on. This has already been implemented and is working.
The `<select>` style must be uniform.

- Changed for the Options page.
- Changed for the Montage page.
- Changed for the Filter page.

On  Montage page more 10 options in select, which is why the search is enabled. If you don’t like this behavior, then the search can be excluded for some specific select elements, but leave the “chosen” style.

### CODE REVIEW

**Comment by IgorA100** [comment]
My work is finished.
I accept comments if something is not satisfactory.

File: web/skins/classic/includes/functions.php [hunk 1]
```diff
@@ -1021,7 +1021,17 @@ function xhtmlFooter() {
   $skinJsFile = getSkinFile('js/skin.js');
 ?>
   <script src="<?php echo cache_bust($skinJsFile) ?>"></script>
-  <script nonce="<?php echo $cspNonce; ?>">$j('.chosen').chosen({disable_search_threshold: 10});</script>
+  <script nonce="<?php echo $cspNonce; ?>">
```
Comment (connortechnology) [inline web/skins/classic/includes/functions.php]: Instead of having this in php, let's stick the function in skin.js

Comment (IgorA100) [reply web/skins/classic/includes/functions.php]: No problem. I thought about it, but decided not to touch it for now.
I don't know what to do with the $cspNonce token yet, but I'll look at the code

Comment (connortechnology) [reply web/skins/classic/includes/functions.php]: remove it. Remove the entire script section.

Comment (IgorA100) [reply web/skins/classic/includes/functions.php]: I transferred script
The $cspNonce token now applies to entire main JS script.
I have no problems with tests.
I haven’t tested the multi-server mode, but in theory there shouldn’t be any problems.

**Pull request merged**